src/main/java/mate/academy/internetshop/controllers/CompleteOrderController.java

```diff
@@ -21,7 +21,8 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        ShoppingCart shoppingCart = shoppingCartService.get(USER_ID).get();
+        Long userId = (Long)req.getSession().getAttribute(USER_ID);
+        ShoppingCart shoppingCart = shoppingCartService.get(userId).get();
         orderService.completeOrder(shoppingCartService.getAllProducts(shoppingCart),
```

Comment:
```suggestion
        orderService.completeOrder(shoppingCart.getProducts(),
```
давай зробимо так щоб не використовувати двічі shoppingCartService